### PR TITLE
feat(assets): add op27 row table

### PR DESCRIPTION
**Pull request by masthom**

### CODE REVIEW

File: src/assets/data/edition/series/1/section/2/Op27/folio-convolute.json
Comment (musicEnfanthen) [inline src/assets/data/edition/series/1/section/2/Op27/folio-convolute.json:1]: Please rename the upper folder to "op27" (small letter).

File: src/assets/data/edition/series/1/section/2/Op27/textcritics.json [hunk 1]
```diff
@@ -0,0 +1,490 @@
+{
+    "textcritics": [
+        {
+            "id": "SkRT",
+            "label": "M 325 Sk# / M 326 Sk1 / M 327 Sk1 (Reihentabelle op. 27)",
+            "description": [
+                "<small class=\"text-muted\">[Der Skizzenkommentar erscheint im Zusammenhang der vollständigen Edition von Opus 24 in AWG I/1.]</small>"
```
Comment (musicEnfanthen) [inline src/assets/data/edition/series/1/section/2/Op27/textcritics.json:7]:
```suggestion
                "<small class=\"text-muted\">[Der Skizzenkommentar erscheint im Zusammenhang der vollständigen Edition von Opus 27 in AWG I/2.]</small>"
```

File: src/assets/img/edition/series/1/section/2/Op27/Op. 27_Reihentabelle_20220120_1von3_path.svg
Comment (musicEnfanthen) [inline src/assets/img/edition/series/1/section/2/Op27/Op. 27_Reihentabelle_20220120_1von3_path.svg:1]: rename complex folder: "op27" (small letter)

rename file: op27_B_Reihentabelle-1von3-final.svg

File: src/assets/img/edition/series/1/section/2/Op27/Op. 27_Reihentabelle_20220120_3von3_path.svg
Comment (musicEnfanthen) [inline src/assets/img/edition/series/1/section/2/Op27/Op. 27_Reihentabelle_20220120_3von3_path.svg:1]: rename file: op27_B_Reihentabelle-3von3-final.svg

File: src/assets/img/edition/series/1/section/2/Op27/Op. 27_Reihentabelle_20220120_2von3_path.svg
Comment (musicEnfanthen) [inline src/assets/img/edition/series/1/section/2/Op27/Op. 27_Reihentabelle_20220120_2von3_path.svg:1]: rename file: op27_B_Reihentabelle-2von3-final.svg

File: src/assets/data/edition/series/1/section/2/Op27/folio-convolute.json
Comment (masthom) [reply src/assets/data/edition/series/1/section/2/Op27/folio-convolute.json:1]: done

File: src/assets/img/edition/series/1/section/2/Op27/Op. 27_Reihentabelle_20220120_1von3_path.svg
Comment (masthom) [reply src/assets/img/edition/series/1/section/2/Op27/Op. 27_Reihentabelle_20220120_1von3_path.svg:1]: done

File: src/assets/img/edition/series/1/section/2/Op27/Op. 27_Reihentabelle_20220120_3von3_path.svg
Comment (masthom) [reply src/assets/img/edition/series/1/section/2/Op27/Op. 27_Reihentabelle_20220120_3von3_path.svg:1]: done

File: src/assets/img/edition/series/1/section/2/Op27/Op. 27_Reihentabelle_20220120_1von3_path.svg
Comment (masthom) [reply src/assets/img/edition/series/1/section/2/Op27/Op. 27_Reihentabelle_20220120_1von3_path.svg:1]: did not work... :-(

**Pull request merged**